### ISSUE TITLE
Removing PCR duplicates before running GenomeScope2/Smudgeplot

### ISSUE THREAD

**Issue by ms-gx** [schatzlab/genomescope#40]
Is it advisable to remove PCR duplicates in an aligment-free way before running GenomeScope2 or Smudeplot?

If no, why not?

If yes, are there any points too keep in mind while doing this?

**Comment by mschatz** [comment]
No, this is generally not necessary except perhaps for extreme levels.
GenomeScope uses a negative binomial model rather than a strict Poisson to
account for the overdispersion that occurs from PCR duplicates. For
Smudgeplot, just make sure to pick your low and high coverage cutoffs to
include the entire peak

Good luck!

Mike

**Comment by ms-gx** [comment]
Thanks Mike!

Please allow me a quick follow-up:

1) I am not completely sure how to interpret the yellow line in the GenomeScope2 coverage plot (as for example in the two plots linked below). My understanding is that this is the model fitted without the repetitive regions of the genome. Is that correct?

2) I run GenomeScope2 on my data i) once as being diploid and ii) once being tetraploid. What would your bet be about the reality of the situation? :)
i) Diploid analysis: http://genomescope.org/genomescope2.0/analysis.php?code=5y4SQw0TTNzI94JjYNI0
ii) Tetraploid analysis: http://genomescope.org/genomescope2.0/analysis.php?code=h4gMKlFJvOwuHm3gOJzJ

I am aware that I can use SmudgePlot to investigate this (kmc actually running right now) but I am curious to hear what you would say from the GenomeScope2 plots...

**Comment by tbenavi1** [comment]
You interpretation of the yellow line is correct. It is somewhat difficult to tell ploidy from the GenomeScope plots based on your data, but I would guess that it is diploid. SmudgePlot should provide better evidence. Thanks.

**Comment by ms-gx** [comment]
Thank you for your feedback!

Here is the Smudgeplot: https://github.com/KamilSJaron/smudgeplot/issues/62

What would you say in combination with that?
